core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -110,6 +110,7 @@
     public static final String KEY_INCLUDE_FILTER = "include_filter";
     public static final String KEY_EXCLUDE_FILTER = "exclude_filter";
     public static final String KEY_FEED_PLAYBACK_SPEED = "feed_playback_speed";
+    public static final String KEY_INITIATED_BY_USER = "initiated_by_user";
```

Comment:
I do not think this is needed

Reply:
It's used in `DownloadStatus` to create a staus from a cursor.

Reply:
The cursor is used for things that come from the database. You did not update the database to contain this column. I think when creating a status from cursor, the initiatedByUser field does not matter (it is just used for the Downloads»Logs page). So you can just set it to false and remove the key here.

Reply:
OK that makes sense, changed.